### PR TITLE
Make test setup more flexible

### PR DESCRIPTION
**Pull request by mar-kolya**
This change allows submodules to set `forceJdk` property to an array of JVM names that this module would want tests to be run on. For example setting that to `IBM8` would run tests on `IBM JVM` even if they otherwise would not (because for example they are java11 only).

### CODE REVIEW

File: gradle/java.gradle [hunk 1]
```diff
@@ -219,6 +219,10 @@ def isJavaVersionAllowed(JavaVersion version) {
   return true
 }
 
+def isJdkForced(String javaName) {
+  return (project.hasProperty('forceJdk') && project.getProperty('forceJdk').contains(javaName))
```
Comment (tylerbenson) [inline gradle/java.gradle]: Why is this needed?

Comment (mar-kolya) [reply gradle/java.gradle]: Sometimes it is useful to run things on JVM that would be disabled bu default
